mlflow/models/signature.py

```diff
@@ -70,6 +73,8 @@ def __init__(self, inputs: Schema, outputs: Schema = None, params: ParamSchema =
                 "If params are provided, they must by of type mlflow.models.signature.ParamSchema. "
                 f"Got '{type(params)}'"
             )
+        if all(x is None for x in [inputs, outputs, params]):
```

Comment:
/ai can we use `[inputs, outputs, params]`.count(None) here? Is it faster or slower?

Reply:
/ai Thanks! How would you refactor this line?